shopify/resources/variant.py

```diff
@@ -8,12 +8,19 @@ class Variant(ShopifyResource, mixins.Metafields):
     @classmethod
     def _prefix(cls, options={}):
         product_id = options.get("product_id")
-        if product_id:
+        if options.get('is_new'):
```

Comment:
Does this actually fix something?  This PR is very hard to review because of these scattered changes without much context.